scripts/RunLiveTests-AWSPowershell.ps1

```diff
@@ -16,16 +16,13 @@ param (
     $IsVpcStack
 )
 
-if ("$(imageReleaseState)" -eq "Production") {
+if ($imageReleaseState -eq "Production") {
     $SkuName = "$BaseImageName-$(TestVersionPrev-$BaseImageName)"
```

Comment:
This code will not work. Its trying to concatenate `$BaseImageName` onto the text `TestVersionPrev-` in order to obtain a variable name like `$TestVersionPrev-w16d-15-0` which is then resolved to a variable value in Azure DevOps. Here are all the variable names that its trying to resolve:
![image](https://user-images.githubusercontent.com/4089160/124849524-48af8f00-dfe2-11eb-8f0b-d9c1986064f0.png)

I do not know of a way to do this in Powershell. Do you? It would be neat, but I think we need to pass the actual value in. That is, pass in the `SkuName`, like `AlternateImageVersionTest.ps1`